packages/storage-events/src/replay.spec.js

```diff
@@ -217,6 +217,38 @@ describe("replay", () => {
         expect(state.todos).toMatchSnapshot();
       });
 
+      it("works happily", () => {
```

Comment:
Please a different test description (what about "can change the title of a todo"?)

Reply:
yup, just didn't know what was wrong in the first place and therefore didn't know what it tests :)